### DISCUSSION TITLE
Row Grouping

### DISCUSSION THREAD

**Discussion by davidmcooper** [glideapps/glide-data-grid#814] [Q&A]
Are there any plans to add row grouping support to this grid to enable hierarchical presentation of data that can be dynamically collapsed and expanded?

**Comment by KyDenZ** [comment]
He has an open MR on this subject:

https://github.com/glideapps/glide-data-grid/pull/358

I'm really waiting for her to be merged to use this project.

**Reply by jassmith** [reply]
This is in 6.0.0 in a different form
